db/osqlcomm.c

```diff
@@ -1115,38 +1162,32 @@ static uint8_t *osqlcomm_done_rpl_put(const osql_done_rpl_t *p_osql_done_rpl,
     return p_buf;
 }
 
-static const uint8_t *osqlcomm_done_rpl_get(osql_done_rpl_t *p_osql_done_rpl,
-                                            const uint8_t *p_buf,
-                                            const uint8_t *p_buf_end)
-{
-    if (p_buf_end < p_buf || OSQLCOMM_DONE_RPL_LEN > (p_buf_end - p_buf))
-        return NULL;
-
-    p_buf = osqlcomm_rpl_type_get(&(p_osql_done_rpl->hd), p_buf, p_buf_end);
-    p_buf = osqlcomm_done_type_get(&(p_osql_done_rpl->dt), p_buf, p_buf_end);
-
-    return p_buf;
-}
-
 typedef struct osql_done_uuid_rpl {
     osql_uuid_rpl_t hd;
     osql_done_t dt;
+    struct query_effects effects;
+    struct query_effects fk_effects;
 } osql_done_uuid_rpl_t;
 
 enum {
-    OSQLCOMM_DONE_UUID_RPL_LEN =
-        OSQLCOMM_UUID_RPL_TYPE_LEN + OSQLCOMM_DONE_TYPE_LEN
+    OSQLCOMM_DONE_UUID_RPL_v1_LEN =
+        OSQLCOMM_UUID_RPL_TYPE_LEN + OSQLCOMM_DONE_TYPE_LEN,
+    OSQLCOMM_DONE_UUID_RPL_v2_LEN =
+        OSQLCOMM_DONE_UUID_RPL_v1_LEN + (2 * sizeof(struct query_effects)),
 };
 
+#if 0
```

Comment:
why if 0 this one?

Reply:
I was not sure whether to keep it or not.